packages/manager/modules/nutanix/src/service.js

```diff
@@ -304,4 +304,18 @@ export default class NutanixService {
       ),
     };
   }
+
+  updateDedicatedNodePowerStateOn(nodeId) {
+    return this.$http
+      .put(`/dedicated/server/${nodeId}`, {
+        bootId: 1,
+        monitoring: false,
+        noIntervention: false,
+      })
+      .then(() => this.rebootDedicatedNode(nodeId));
+  }
+
+  rebootDedicatedNode(nodeId) {
+    return this.$http.post(`/dedicated/server/${nodeId}/reboot`);
```

Comment:
Could reboot be called in a standalone mode ? If not you should put it directly in 'powerStateOn' function

Reply:
Not but I need this same call for the poweroff function